.github/workflows/issue-comment-created.yaml

```diff
@@ -7,8 +7,6 @@ on:
 jobs:
   issue_comment_triage:
     runs-on: ubuntu-latest
-    permissions:
-      pull-requests: write
```

Comment:
we need to explicitly define the permissions being used here, since the default token will be changing to read shortly